website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md

```diff
@@ -42,27 +44,38 @@ Back then, it all sort of made sense. You needed software and systems that could
 
 **What has changed and how is that changing the data layer?**
 
-![https://cdn-images-1.medium.com/max/800/0*ZJ8CaKQdxEesL7eE](https://lh3.googleusercontent.com/I8EEDuFVhtL2yoa5VUjoOOGvAO5X7o5miPjxV3fe4rombAU9Gz6ZA7sD-C3Alw8fRVpyMzCqW-OqCsGib5RyW0pi-7xJnZpdruVdGKMDeuU-P9sca_2XY6X1itGjMREag0MoLj8D)
+![https://cdn-images-1.medium.com/max/800/0*ZJ8CaKQdxEesL7eE](https://cdn-images-1.medium.com/max/800/0*ZJ8CaKQdxEesL7eE)
```

Comment:
Let's try to understand what the image says and accordingly let's set the alt text for that image.